packages/insomnia/src/common/database.ts

```diff
@@ -709,6 +709,11 @@ type Patch<T> = Partial<T>;
 // Helpers //
 // ~~~~~~~ //
 async function _send<T>(fnName: string, ...args: any[]) {
+  if (['update', 'insert'].includes(fnName) && args[0] && !['Stats', 'Settings', 'PluginData', 'GitRepository'].includes(args[0].type) && !args[0].parentId) {
```

Comment:
```suggestion
  if (['update', 'insert'].includes(fnName) && args[0] && args[0].type==='Workspace' && !args[0].parentId) {
```
perhaps just track the thing we know goes missing?

Reply:
agreed

Reply:
project parent id seems to go missing, too, and there is no way for project parent id to be null any more